package.json

```
@@ -12,7 +12,8 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "tsc --allowSyntheticDefaultImports",
+    "build:js": "npm run build && rollup -c",
```

This project uses `yarn` not `npm`. Please update to:
```suggestion
    "build:js": "yarn build && rollup -c",
```